types/inet.go

```diff
@@ -154,6 +149,13 @@ func (a *InetArray) Value(i int) *net.IPNet {
 	if a.IsNull(i) {
 		return nil
 	}
+	cidr := string(a.Storage().(*array.Binary).Value(i))
+	if len(cidr) == 0 {
+		return &net.IPNet{
+			IP:   net.IPv4zero,
+			Mask: make(net.IPMask, 4),
+		}
+	}
 	_, ipnet, err := net.ParseCIDR(string(a.Storage().(*array.Binary).Value(i)))
```

Comment:
seems you can call `cidr` here you call `(a.Storage().(*array.Binary).Value(i))` before that

Reply:
Done in a8076df52d738719811bd89c894f3c2a11f1f512